diff --git a/lib/internal/streams/destroy.js b/lib/internal/streams/destroy.js
index 38292315ee9ae159634fc0646faa57bd46088a6f..1e5e81db24b2b91f8a2766f7e5929cb81ebeded6 100644
--- a/lib/internal/streams/destroy.js
+++ b/lib/internal/streams/destroy.js
@@ -2,7 +2,7 @@
 
 /* replacement start */
 
-const process = require('process/')
+const process = require('process')
 
 /* replacement end */
 
diff --git a/lib/internal/streams/duplexify.js b/lib/internal/streams/duplexify.js
index 05740d70ff018aae02fceb6ffd0ee669f1ef6c1c..46fdecbd5e02274977c79d76a0a9d0554c7ce737 100644
--- a/lib/internal/streams/duplexify.js
+++ b/lib/internal/streams/duplexify.js
@@ -1,6 +1,6 @@
 /* replacement start */
 
-const process = require('process/')
+const process = require('process')
 
 /* replacement end */
 
diff --git a/lib/internal/streams/end-of-stream.js b/lib/internal/streams/end-of-stream.js
index 94d18321d29026450ce0b0fdff88875e7e82281d..92b8aeadca3764416639b67eab4b1abf4cb6aca7 100644
--- a/lib/internal/streams/end-of-stream.js
+++ b/lib/internal/streams/end-of-stream.js
@@ -5,7 +5,7 @@
 
 /* replacement start */
 
-const process = require('process/')
+const process = require('process')
 
 /* replacement end */
 
diff --git a/lib/internal/streams/from.js b/lib/internal/streams/from.js
index c7e75314028794c050b62171a07b654366203c88..2b3a8411bbd4f7df606f43340e18354bdb6cb154 100644
--- a/lib/internal/streams/from.js
+++ b/lib/internal/streams/from.js
@@ -2,7 +2,7 @@
 
 /* replacement start */
 
-const process = require('process/')
+const process = require('process')
 
 /* replacement end */
 
diff --git a/lib/internal/streams/pipeline.js b/lib/internal/streams/pipeline.js
index a2bab880069748ab2b1ab0a0ffa9426fbfabce47..a6eaa23f0b118bd880cde6aac4b8c495129e5dba 100644
--- a/lib/internal/streams/pipeline.js
+++ b/lib/internal/streams/pipeline.js
@@ -1,6 +1,6 @@
 /* replacement start */
 
-const process = require('process/')
+const process = require('process')
 
 /* replacement end */
 // Ported from https://github.com/mafintosh/pump with
diff --git a/lib/internal/streams/readable.js b/lib/internal/streams/readable.js
index 90c7316056a73edb694a7e7b54fe2d772f3c8a60..5b293f1372fdda78793424562095231a6415e908 100644
--- a/lib/internal/streams/readable.js
+++ b/lib/internal/streams/readable.js
@@ -23,7 +23,7 @@
 
 /* replacement start */
 
-const process = require('process/')
+const process = require('process')
 
 /* replacement end */
 
diff --git a/lib/internal/streams/writable.js b/lib/internal/streams/writable.js
index b4ecf0e21dd929fb374f3b0af31ad2d5ecdf04ca..9f4c2f8943c3a627d1c5820221adca7ef9639329 100644
--- a/lib/internal/streams/writable.js
+++ b/lib/internal/streams/writable.js
@@ -27,7 +27,7 @@
 
 /* replacement start */
 
-const process = require('process/')
+const process = require('process')
 
 /* replacement end */
 
